### PR TITLE
Update: CSS grid for placards, colors to meet a11y guidelines

### PR DESCRIPTION
**Pull request by schwigri**

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -71,10 +71,47 @@ form label {
...
+.btn-primary {
+    background-color: #0071eb;
+    border-color: #0071eb;
+}
+
+.btn-primary.disabled,
+.btn-primary:disabled {
+    background-color: #eee;
+    border-color: #ddd;
+    color: #696969;
```
Comment (evelynhathaway) [inline public/style.css:87]: Noice 😏👌

File: public/style.css [hunk 2]
```diff
@@ -71,10 +71,47 @@ form label {
...
+    opacity: 1;
+}
+
+.placards {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
```
Comment (evelynhathaway) [inline public/style.css]: Are there mobile styles for the grid?

File: public/style.css [hunk 3]
```diff
@@ -71,10 +71,47 @@ form label {
...
+    -ms-flex-direction: column;
+    flex-direction: column;
+    justify-content: center;
+    margin: 1rem 0;
+    min-width: 0;
+    word-wrap: break-word;
+    background-clip: border-box;
+    border: 1px solid rgba(0,0,0,.125);
+    border-radius: .25rem;
+}
```
Comment (evelynhathaway) [inline public/style.css:113]: I really like this take on the empty state. 😄👍

**Review by evelynhathaway — COMMENTED** [review]
Thanks for submitting this! Just one question/change before it's good to go. 😄

File: public/style.css (hunk 2 above)
Comment (schwigri) [reply public/style.css]: This is good idea! I'll see if I can add this.

Comment (schwigri) [reply public/style.css]: I've just added responsive grid styles ☺️

**Review by evelynhathaway — APPROVED** [review]
Looks awesome! Thank you for your contribution, Griffen!

**Pull request merged**